src/server/middlewares/authentication.js

```diff
@@ -20,7 +20,6 @@ const Auth = (req, res, next) => {
         message: 'Invalid token provided',
       });
     }
-
     res.locals.currentUser = decoded;
```

Comment:
Can we have something like `req.headers.userid=decoded;`. So that anywhere this authentication is used, we can access the `userid` directly from the header instead of having to decode all over again.

Reply:
Well `res.locals.currentUser ` is already serving that if we want userId, all that needs to be done is `res.locals.currentUser.id`